frontend/app/views/spree/checkout/_payment.html.erb

```diff
@@ -59,7 +59,15 @@
     <p class='field' data-hook='coupon_code'>
       <%= form.label :coupon_code %><br />
       <%= form.text_field :coupon_code %>
+      <button type="button" class="button" id="coupon-code-apply-button">
+        <%= Spree.t(:apply_code) %>
+      </button>
     </p>
+    <div id='coupon_status'
+      data-success-class="success"
```

Comment:
Yes, but they are present for documentation purposes here. Alternatively, I can remove the defaults in the JS, but I thought they don't add much noise, so why not have them.